test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
Good call with disabling speed stepping, I hadn't thought of that.

New GH issue: https://github.com/tumblr/collins/issues/550